src/components/CandidateContext.tsx

```diff
@@ -1,6 +1,11 @@
 import { createContext } from 'react'
 
-const contextObj = {
```

Comment:
I don't think this is the optimal use case for using the Context API in react, but it works.